src/machine/machinectl.c

```diff
@@ -1616,7 +1617,7 @@ static int enable_machine(int argc, char *argv[], void *userdata) {
         if (r < 0)
                 return bus_log_create_error(r);
 
-        for (int i = 1; i < argc; i++) {
```

Comment:
why is this being moved? we do want to declare variables in the closest scope possible